lib/remote/jsonrpcconnection.hpp

```diff
@@ -20,6 +20,8 @@
 namespace icinga
 {
 
+using namespace std::chrono_literals;
```

Comment:
The last time I've used this language construct inside a 3-line anonymous function(!), @julianbrost insisted on `namespace fs=`. You're using this in a namespace in a header!

1. Remove this line.
2. Optionally, given one could see `std::chrono_literals` as special case like `boost::placeholders`: Move `using namespace std::chrono_literals;` from httpserverconnection.hpp to, say, i2-base.hpp if this usage eases things so much.